react/typings/global.d.ts

```diff
@@ -3,566 +3,288 @@ import { NormalizedCacheObject } from 'apollo-cache-inmemory'
 import { EventEmitter } from 'eventemitter3'
 import { Component } from 'react'
 import ExtensionContainer from '../ExtensionContainer'
-import ExtensionPoint from '../ExtensionPoint'
+import ExtensionPoint from '../components/ExtensionPoint/index'
```

Comment:
I'm all up for explicitness, but is this `/index` needed?

Reply:
Yes, there's another component at `../components/ExtensionPoint.tsx` so I can export its types